tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{39,310,311}-test{,-alldeps,-shapely,-specutils}{,-cov}
-    py{39,310,311}-test-numpy{120,121,122,123}
-    py{39,310,311}-test-{numpy,astropy,linetools,ginga}dev
+    {3.9,3.10,3.11}-test{,-alldeps,-shapely,-specutils}{,-cov}
+    {3.9,3.10,3.11}-test-numpy{120,121,122,123}
```

Comment:
Since we require NumPy>=1.22 (`setup.cfg`) and newer versions are out, should we amend this to `{122,123,124,125}`?

Reply:
yes! nice catch...